wqflask/wqflask/templates/snp_browser.html

```diff
@@ -508,6 +524,39 @@ <h2>Variant Browser <a class="btn btn-primary" href="http://genenetwork.org/snpb
       });
       $("input[name=chosen_strains]").val(strain_list.join(","));
     });
+
+    $("input[name=export_csv]").click(function() {
+      var csv = [];
+      var rows = document.querySelectorAll("table tr");
+
+      var headers = [];
+      var col_header = rows[0].querySelectorAll("th");
+      for(var i = 1; i < col_header.length; i++) {
+        headers.push(col_header[i].getAttribute("name"));
+      }
+      csv.push(headers.join(","));
+
+      for (var i = 1; i < rows.length; i++) {
+        var row = [], cols = rows[i].querySelectorAll("td");
+        var checkBox = rows[i].querySelector("input");
+        console.log(checkBox);
+
+        if(checkBox.checked == true) {
+          for (var j = 1; j < cols.length; j++)
+            row.push(cols[j].innerText);
+
+            csv.push(row.join(","));
+        }
+      }
+
+      var csvFile = new Blob([csv.join("\n")], {type: "text/csv"});
+      var downloadLink = document.createElement("a");
+      downloadLink.download = "variant_data.csv";
+      downloadLink.href = window.URL.createObjectURL(csvFile);
+      downloadLink.style.display = "none";
+      document.body.appendChild(downloadLink);
+      downloadLink.click();
```

Comment:
**Chore:** Use `let` instead of `var`
I think it's better to use the `let` form to limit the scope of variables and to avoid conflicts. Also, it's well supported in alot of modern browsers so I don't think it'd be a problem.

**Suggestion:** Make our querySelectors less generic(and less ambiguous)
I can see: `var rows = document.querySelectorAll("table tr");` and `rows[i].querySelectorAll("td");`, etc etc. It'd be a good idea to make the css more specific e.g. `table.className table tr` or even`classNameForRow`. That way, were another table was added later, there wouldn't be any conflict.
```

Reply:
I read about the difference. Yes `let` is better to use due to scope of variables. I'll change that in the next commit. Thanks for letting me know.